.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
Do you think we can have it monthly ? I'm scared of new PR's coming daily!

Reply:
Yeah, you are right. `monthly` is a better option! :)